api-tests/management/postgresql_test.go

```diff
@@ -100,14 +101,15 @@ func TestAddPostgreSQL(t *testing.T) {
 		assert.Equal(t, agents.ListAgentsOKBody{
 			PostgresExporter: []*agents.ListAgentsOKBodyPostgresExporterItems0{
 				{
-					AgentID:            listAgents.Payload.PostgresExporter[0].AgentID,
-					ServiceID:          serviceID,
-					PMMAgentID:         pmmAgentID,
-					Username:           "username",
-					DisabledCollectors: []string{"custom_query.ml", "custom_query.mr.directory"},
-					PushMetricsEnabled: true,
-					Status:             &AgentStatusUnknown,
-					AutoDiscoveryLimit: 10,
+					AgentID:                listAgents.Payload.PostgresExporter[0].AgentID,
+					ServiceID:              serviceID,
+					PMMAgentID:             pmmAgentID,
+					Username:               "username",
+					DisabledCollectors:     []string{"custom_query.ml", "custom_query.mr.directory"},
+					PushMetricsEnabled:     true,
+					Status:                 &AgentStatusUnknown,
+					AutoDiscoveryLimit:     10,
+					MaxExporterConnections: 0,
```

Comment:
seems like behaviour of AutoDiscoveryLimit and MaxExporterConnections are different, can we have them consistent?

Reply:
I mean that above you added both items with value = `0`, but in response they have different values.

Reply:
Ah... with `max_exporter_connections` logic is that if it's not set, then it's up to exporter to decide (i.e. default value set on the exporter side). Not sure that it makes sense to have it on PMM side. With current logic we can easily change default value without DB migration.

Reply:
Probably we still need it until we finish the last part.
We could achieve `server-defined` result without storing the value in DB, but having it hardcoded.

Reply:
After discussion we decided to change `AutoDiscoveryLimit` behavior instead.